src/components/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { ReactElement } from "react";
+import '.././styles.css';
+
+const Footer = (): ReactElement => {
+  return (
+    <div className="w-full px-2 py-4 mt-4 flex flex-row justify-between border-t-2 absolute bottom-0">
+      <div>
+        Made by students @{" "}
+        <a className="link"
+          onClick={() => {
```

Comment:
Is it possible to use `href` here instead of `window.open`? I'm more familiar with links utilizing `href`, and I'm wondering if that might be faster (since this `window.open` utilizes JS)

Reply:
So I think window.open() opens the link in a new window, which we might not need for all the links in our app, but I talked to Melody and she said it'd be good for footer links to open in a new window.

Reply:
Ooh gotcha, that makes sense